### ISSUE TITLE
Some typos I found while reading through the docs

### ISSUE THREAD

**Issue by fwitte** [chmarti1/pyrodoc#4]
Hi @chmarti1 and @jranalli,

- The link to the handbook is broken https://github.com/chmarti1/pyrodoc/blob/4d2040472e337403030ea3198ebf998bcd9a8052/html/about.html#L163 and https://github.com/chmarti1/pyrodoc/blob/4d2040472e337403030ea3198ebf998bcd9a8052/html/about.html#L172 It should be `pdf/handbook.pdf`
- These (1-5) could be actual bullets for better readability and (5) is the best reason I can imagine ;). https://github.com/chmarti1/pyrodoc/blob/4d2040472e337403030ea3198ebf998bcd9a8052/html/about.html#L166
- Some code blocks are outside of the paragraph-blocks https://github.com/chmarti1/pyrodoc/blob/4d2040472e337403030ea3198ebf998bcd9a8052/html/download.html#L74-L78 some inside https://github.com/chmarti1/pyrodoc/blob/4d2040472e337403030ea3198ebf998bcd9a8052/html/download.html#L65-L71 Is that intentional? 
- In English, do you actually say `heated to a boil` (https://github.com/chmarti1/pyrodoc/blob/4d2040472e337403030ea3198ebf998bcd9a8052/html/live/rankine.html#L76)? Or is the `a`superfluous?
- In the superheated Rankine cycle demo, work of turbines is calculated with inverted sign convention to heat transferred in the boiler, superheater and condenser. Is that intentional? See https://github.com/chmarti1/PYroMat/commit/370f0b8db2c3dae9d30b29aab67a74f14adcab9b#diff-9e1ae1a2737274e9fd787a625ea5510ed6be5a6bd9b0360186ef70fde3ff6a1aL890-L902 (I guess that is the correct place)
- The `tp` keyword does not work anymore https://github.com/chmarti1/pyrodoc/blob/4d2040472e337403030ea3198ebf998bcd9a8052/html/doc_howto.html#L153-L154

Because I am curious: Why do you leave double spaces after dots in sentences? Something to search-replace and figure out if it is a floating point or abbreviation?

**Comment by jranalli** [comment]
> * In English, do you actually say `heated to a boil` 

I will let Chris comment on most of these things, since it's his writing, but I did want to hit this one, because these minutiae of language are one of my favorite things, especially given the global audience for english these days.

Short answer:
This is a typical way to phrase this, at least in American English.

Longer Answer:
This appears to be a location and context specific usage. I am from the east coast of the US. I would give no pause to the phrases `heated to a boil`, `heated to boil` or `heated to boiling`. Whether they're "proper English" or not, I can't say, but they at least sound natural enough to me as a native American English speaker that I would not object to any of them spoken or in writing. But we should note that most of our everyday usage of these terms relates to cooking. So if I were myself telling you how to cook pasta, I would say: 
`Add the water to the pot, bring it to a boil, then add the noodles.` 
The instructions on boxes of pasta and oatmeal in my pantry agree with that `to a boil` usage, and I would say that's probably the most common way to phrase this in American English.

British English apparently uses a phrasing that I have never heard before today: [`bring to the boil`](https://english.stackexchange.com/questions/296256/bring-to-a-boil-vs-bring-to-the-boil). 

In a technical article or something, I would say that "bring to a boil" is probably not the best term, since I think of "boil" as loosely defined. So textbooks on thermodynamics would likely to use more precise terminology like "convert to steam" or "vaporize" or "bring to the boiling point/saturation temperature" or something like that. 

So in sum, if you're talking with some American friends about making some tea, you might talk about bringing the water to A boil, otherwise, it's probably better to use more scientific language to describe the process. But if there's any larger conclusion we can reach, I think it's that English is a messy, messy language!

**Comment by chmarti1** [comment]
First off, thanks so much for the detailed comments!  It's rare we get detailed feedback on the documentation.

(1) I'll be correcting those broken links today.  Good catch!
(2) I think you're right - an ordered list looks tidier, and it reads better.  (It's great to hear someone else thinks the fun is part of the point!)
(3) I originally had a principled idea for the use of code blocks/paragraphs, but it looks like I got sloppy in this page.  Now that you've drawn attention to it, I don't like the inconsistent spacing.  I'll fix that.
(4) @jranalli is right, "heat to a boil" is more commonly used when cooking.  I am going to elect not to update the live pages, since those are slated to be completely replaced soon.
(5) Great catch on the `tp=True` keyword in the saturation methods.  In the original algorithm for saturation properties this used to make sense, but it was dropped quite some time ago in a major revision release.  I'd trimmed it from the handbook, but looks like I missed it in the "how-to" section.  Thanks!

While digging through, I found a few typos of my own.  I'll push these out today.  Thanks again!
